src/lib.rs

```diff
@@ -280,45 +231,43 @@ mod tests {
 
     #[test]
     fn eq_mutation() {
-        let mu = Mutagen { x: AtomicUsize::new(0) };
-
+        reset();
```

Comment:
I think we can get rid of `reset` and `next` using:
```
MU.x.store(n, Ordering::SeqCst)
```

Reply:
Possibly, but `reset` and `next` make the meaning more clear, I think.